src/components/containers/ContainerEditContent.js

```diff
@@ -164,21 +164,21 @@ const ContainerEditContent = ({token, containerKinds, containersByID, add, updat
             <TextArea />
           </Item>
           {formErrors?.non_field_errors &&
-            <>
-              <Alert
-                showIcon
-                type="error"
-                message="Error(s)"
-                description={
-                  <ul>
-                    {formErrors.non_field_errors.map((e, i) =>
-                      <li key={i}>{e}</li>
-                    )}
-                  </ul>
-                }
-              />
-              <br/>
-            </>
+            <Alert
+              showIcon
+              type="error"
+              style={{ marginBottom: '1em' }}
```

Comment:
I understand the concept of displaying the alert 'in the face' of the user, but I think it's not common to have it at the bottom of the page like this. Maybe at the top as a sticky warning that can be closed (x) ? What do you think?